src/LibraryManager/Configuration/Settings.cs

```diff
@@ -64,24 +64,18 @@ protected Settings()
                 }
                 catch (JsonReaderException)
                 {
-                    // If there were any errors, we'll reset the file
+                    // If there were any errors, we'll take the defaults
+                    // TODO: log a warning
                 }
             }
 
             if (_rootObject is null)
             {
-                InitSettingsFile(ConfigFilePath);
+                _rootObject = new JObject(new JProperty("config", new JObject()));
```

Comment:
I know it's not the part you are changing, but what do you think of adding a couple of retries for reading the existing file before giving up and going to the empty defaults? If it's for stuff like proxy configuration, it seems like existing data is pretty important :) Not sure why we decided to overwrite with empty file on first failure to begin with.

Reply:
The original idea was that if the file is corrupted (e.g. invalid JSON), then we'd fall back to the defaults (and reset the file).    Retries won't fix a file that doesn't exist or isn't parseable JSON.  For this issue, the problem was caused by a lingering write lock.  We could retry until that lock is released, but the only time writes occur to the file outside of this (now-removed) reset was by using the CLI (`libman config ...`) or manually editing the file.  Neither of these is likely to overlap with a restore operation in a separate process to cause the same issue.